subgraphs/uniswap-forks/protocols/biswap/config/deployments/biswap-bsc/configurations.ts

```diff
@@ -0,0 +1,120 @@
+import { Address, BigDecimal, BigInt } from "@graphprotocol/graph-ts";
+import { Factory } from "../../../../../generated/Factory/Factory";
+import {
+  BIGINT_ZERO,
+  FeeSwitch,
+  MINIMUM_LIQUIDITY_FIVE_THOUSAND,
+  MINIMUM_LIQUIDITY_TEN_THOUSAND,
+  Network,
+  PROTOCOL_SCHEMA_VERSION,
+  RewardIntervalType,
+} from "../../../../../src/common/constants";
+import { Configurations } from "../../../../../configurations/configurations/interface";
+import {
+  PROTOCOL_SUBGRAPH_VERSION,
+  PROTOCOL_METHODOLOGY_VERSION,
+  PROTOCOL_NAME,
+  PROTOCOL_SLUG,
+} from "../../../src/common/constants";
+
+export class BiswapBscConfigurations implements Configurations {
+  getNetwork(): string {
+    return Network.BSC;
+  }
+  getSchemaVersion(): string {
+    return PROTOCOL_SCHEMA_VERSION;
+  }
+  getSubgraphVersion(): string {
+    return PROTOCOL_SUBGRAPH_VERSION;
+  }
+  getMethodologyVersion(): string {
+    return PROTOCOL_METHODOLOGY_VERSION;
+  }
+  getProtocolName(): string {
+    return PROTOCOL_NAME;
+  }
+  getProtocolSlug(): string {
+    return PROTOCOL_SLUG;
+  }
+  getFactoryAddress(): string {
+    return "0x858e3312ed3a876947ea49d572a7c42de08af7ee";
+  }
+  getFactoryContract(): Factory {
+    return Factory.bind(
+      Address.fromString("0x858e3312ed3a876947ea49d572a7c42de08af7ee")
+    );
+  }
+  getTradeFee(): BigDecimal {
+    return BigDecimal.fromString("0.2");
+  }
+  getProtocolFeeToOn(): BigDecimal {
+    return BigDecimal.fromString("0.05");
+  }
+  getLPFeeToOn(): BigDecimal {
+    return BigDecimal.fromString("0.15");
+  }
+  getProtocolFeeToOff(): BigDecimal {
+    return BigDecimal.fromString("0");
+  }
+  getLPFeeToOff(): BigDecimal {
+    return BigDecimal.fromString("0");
+  }
+  getFeeOnOff(): string {
```

Comment:
The protocol is collecting some of the fees since the start right?

Reply:
I am still working on getting confirmation from biswap team/community on some things.

Example. The initial trading fees were 0.1% (no idea about the allocation of that 0.1% towards protocol or LP fees) and it was then updated to 0.2%.